src/Srclib/Types.hs

```diff
@@ -313,6 +321,8 @@ instance FromJSON LicenseUnitMatchData where
 data SourceUnit = SourceUnit
   { sourceUnitName :: Text
   , sourceUnitType :: Text
+  , sourceUnitTitle :: Maybe Text
+  -- ^ the title of a custom license
```

Comment:
What happens if title is provided to a non-custom-license source unit? Should this actually be a distinct type?

Reply:
Sorry, this was a remnant that I meant to delete. The title field already exists on the LicenseSourceUnit and does not need to be added here. Thanks for pointing it out!